src/pages/product.js

```diff
@@ -1,23 +1,29 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { observer, inject } from "mobx-react";
-import Typography from "material-ui/Typography";
-
 import withData from "lib/apollo/withData";
 import withRoot from "lib/theme/withRoot";
 import withShop from "containers/shop/withShop";
 import Layout from "components/Layout";
+import ProductDetailComponent from "components/ProductDetail";
 
 @withData
 @withRoot
 @withShop
 @inject("uiStore")
 @observer
 class ProductDetail extends Component {
```

Comment:
Should we rename this component to be `ProductDetailPage` and rename the `ProductDetail` import